library/src/webapp/editor/elfinder/sakai/elfinder.html

```diff
@@ -12,12 +12,14 @@
   <link rel="stylesheet" type="text/css" href="/library/webjars/elfinder/2.1.49/css/elfinder.min.css">
   <link rel="stylesheet" type="text/css" href="/library/webjars/elfinder/2.1.49/css/theme.css">
 
-  <!-- Moono Theme CSS -->
+  <!-- Moono Theme CSS was removed in SAK-42168, looks like is not maintained anymore -->
```

Comment:
not sure why this message is needed! can we just remove all of this?

Reply:
ok, done